### ISSUE TITLE
Applepay is not supported

### ISSUE THREAD

**Issue by rajeev-k-tomy** [magmodules/mollie-magento2-hyva-react-checkout#2]
I can see that Applepay is not integrated the current release and there is an open pull request for apple pay.

I integreated that work in custom and tried to test this. However, i have couple of problems.

1. I can see that in the luma checkout, the checkout will be first redirected to the mollie interface. However, this is not happening in the open pull request solution.
2. When I tried to test this in an iphone, I get following error in the log

    ```
       main.ERROR: [2023-09-14T09:37:04+0000] Error executing API call (400: Bad Request): This API endpoint is only available in live mode and cannot be accessed in test mode.. Field: testmode. Documentation: https://docs.mollie.com/overview/handling-errors

       GraphQL (6:5)
       5:   ) {
       6:     mollieApplePayValidation(
              ^
       7:       domain: $domain
    ```
As the error points out, this graphql endpoint is not supported for testing purpose? Not sure what else should be done in order to test this?

A help would be appreciated. Thank you.

**Comment by rajeev-k-tomy** [comment]
It seems we can test this only in production mode. I was able to perform an order with the production environment.

**Comment by Frank-Magmodules** [comment]
Certainly, @rajeev-k-tomy, Apple Pay is exclusively accessible in LIVE mode. If this issue is no longer an issue, kindly consider closing the matter. Appreciate it!
